django/db/backends/base/base.py

```diff
@@ -286,15 +307,35 @@ def get_autocommit(self):
         self.ensure_connection()
         return self.autocommit
 
-    def set_autocommit(self, autocommit):
+    def set_autocommit(self, autocommit, force_begin_transaction_with_broken_autocommit=False):
```

Comment:
I've updated the docstring and changed the text a bit with . I still haven't figured out a better name for the parameter or how to make it obvious. It is working around an non-obvious bug, it took me quite a while just to understand what was going on... :/